agent/command/shell.go

```diff
@@ -117,13 +117,13 @@ func (c *shellExec) Execute(ctx context.Context, _ client.Communicator, logger c
 	logger.Execution().Debug("Preparing script...")
 
 	var err error
-	var originalScript string
-	if err = c.doExpansions(conf.Expansions); err != nil {
+	originalScript := c.Script
+	if err := c.doExpansions(conf.Expansions); err != nil {
 		logger.Execution().Warning(err.Error())
 		return errors.WithStack(err)
 	}
 	if originalScript != c.Script {
-		logger.Task().Info("`${SHELL_EXPANSION}` syntax will only work for Evergreen expansions. See Evergreen FAQ for details.")
+		logger.Task().Info("`${SHELL_EXPANSION}` syntax will only work for Evergreen expansions. For bash expansions, try using 'sed' instead.")
```

Comment:
I think the `sed` trick only work if they're using backticks to do string substitution, and is likely to confuse people.

Reply:
Should we just drop the suggestion on how to fix it?

Reply:
I think so. We could say something like

> ${SHELL_EXPANSION} will not get passed to the shell, since Evergreen uses this syntax for its own expansions.

Reply:
i think it makes sense to just remove

Reply:
LGTM